src/App.tsx

```diff
@@ -17,6 +17,12 @@ const defaultColors = {
     "--animationButtonSelected": "#484848",
 }
 
+const defaultAnimations = {
+    "animation0": SelectedAnimation.DownwardGravity,
+    "animation1": SelectedAnimation.WobblySwarm,
+    "animation2": SelectedAnimation.BubblePop,
+}
+
```

Comment:
I would consider making this an array. Slightly easier to iterate and can be indexed with 0,1,2.

Reply:
done